### ISSUE TITLE
App disconnects immediately on Bluetooth 4.1 & 4.2

### ISSUE THREAD

**Issue by sbm24** [mitchwongho/ESP-Alerts-for-Android#11]
When i used the app on a 2016 model phone having bluetooth v4.1, it connects to esp and immediatly disconnects. issue is also seen on some other old phones too.
can anyone suggest , what mayb the issue? is there connection time mismatch or something?

**Comment by baaatli** [comment]
Did you find a solution to this?

**Comment by sbm24** [comment]
No, i couldn't find. I move to classic blutooth connection instead of ble
